### ISSUE TITLE
TrackBar control does not support UIA accessibility

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4914]
**Environment**
- Microsoft Windows [Version 10.0.19041.388]
- .NET Core SDK 6.0.100-preview.3.21202.5

**Have you experienced this same bug with .NET Framework?:**
- Yes

**Problem description**
![image](https://user-images.githubusercontent.com/23376742/118262205-3bcc5e80-b4bd-11eb-9c61-28f98a0ee708.png)

Accessibility providers:
[pid:9884,providerId:0x280FE0 Main:Nested [pid:28360,providerId:0x280FE0 Main(parent link):Microsoft: MSAA Proxy (unmanaged:UIAutomationCore.dll)]; Nonclient:Microsoft: Non-Client Proxy (unmanaged:uiautomationcore.dll); Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]

**Expected behavior:**
There should be UIA accessibility provider from WinForms.

**Minimal repro:**
Create WinForms app with TrackBar control and analyze it using Inspect.

**Related item:**
This work item is the sub of #3037

**Comment by John-Qiao** [comment]
Verified with .NET 6.0.100-rc.1.21368.3 from main branch, this issue has been fixed that TrackBar control supports the UIA accessibility now.
![TestResult-4919](https://user-images.githubusercontent.com/45864985/126129166-c9f0aae6-bd07-4f1f-933d-5527ae334d97.png)

**Comment by John-Qiao** [comment]
Verified with .NET 6.0.100-preview.7.21369.19 build, this issue has been fixed that TrackBar control supports the UIA accessibility too.
